scripts/test/Muon/PlottingView_test.py

```diff
@@ -323,13 +331,23 @@ def test_plot_errors_not_in_errors_list(self):
         self.view._set_bounds.assert_called_once_with(self.plot_name)
 
     @mock.patch("mantid.plots.plotfunctions.errorbar")
-    def test_plot_workspace_errors(self, error_bar):
+    #@mock.patch("mantid.plots.plotfunctions.plot")
+    def test_plot_workspace_errors(self, error_bar):#, normal_plot):
+        self.view._add_plotted_line = mock.Mock()
         error_bar.return_value = tuple([[] for i in range(3)])
-        self.view.plot_workspace_errors(self.plot_name, self.mock_workspace)
-        self.assertEquals(error_bar.call_count, 1)
+        mock_line = mock.Mock()
+        mock_line.get_label = mock.Mock(return_value = "test")
+        mock_line.remove = mock.Mock()
+        with mock.patch("mantid.plots.plotfunctions.plot") as normal_plot:
+           normal_plot.return_value = tuple([mock_line])
```

Comment:
Indentation is short by a space in this block.

Reply:
how did it pass flake8?